content/content-strategy/define-user-content-needs/create-actionable-artefacts/journey-maps/diagram.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: blocks/image_content_block
+image: content-strategy/journey-map-example.jpg
+link: content-strategy/journey-map-example.jpg
+reverse: false
+stackPosition: top
+imageAlt: Journey map template
+
+level: 2
+display: 3
+long_description: new-long-description.md
+---
+### [3]Step 3: Visualise the journey
```

Comment:
this section heading should be added to the YAML at the top of the file as `title: Step 3: Visualise the journey` 
The `level` and `display` params in the YAML need to be adjusted to '3' and '2' respectively.

Reply:
Steve fixed